src/ZoweUSSNode.ts

```diff
@@ -119,9 +119,7 @@ export class ZoweUSSNode extends vscode.TreeItem implements IZoweTreeNode {
         try {
             responses.push(await zowe.List.fileList(this.getSession(), this.fullPath));
         } catch (err) {
-            vscode.window.showErrorMessage(localize("getChildren.error.response", "Retrieving response from ")
-                                                    + `zowe.List\n${err}\n`);
-            throw Error(localize("getChildren.error.response", "Retrieving response from ") + `zowe.List\n${err}\n`);
+            await utils.errorHandling(err, this.label, (localize("getChildren.error.response", "Retrieving response from zowe.List")));
```

Comment:
Please forgive me if I missed something very important here.
Question: Is there a reason why we want to localize the API names?

Reply:
This is an existing error message. Maybe the localization part is "Retrieving response from"? I don't know.